app/konnect/dev-portal/developers/dev-gen-creds.md

```diff
@@ -4,40 +4,32 @@ no_version: true
 toc: true
 ---
 
-Generate credentials (API keys) for an application. The API key identifies and authenticates
-the developer application making a call. Use the API key either in the request URL (as a query parameter),
-or in the request header. An API key generated in the {{site.konnect_short_name}}
-Dev Portal is a 32-character string associated with an application.
+A credential, or API key, identifies and authenticates the developer application making a request. Use the API key either in the request URL as a query parameter, or in the request header. 
 
-You can generate multiple credentials and delete as needed.
+Also, any credential can be permanently deleted. See [Delete a credential](#delete-a-credential). 
```

Comment:
```suggestion
You can permanently delete any credential at any time. See [Delete a credential](#delete-a-credential). 
```
I suggest changing this into an action and not a passive phrase, because when I see the phrase "any credential can be permanently deleted", I start wondering about who would delete my credential, or why, or any number of other worries.